lib/ipmi_fru.c

```diff
@@ -4564,7 +4564,7 @@ ipmi_fru_main(struct ipmi_intf * intf, int argc, char ** argv)
 					ipmi_fru_edit_help();
 					return -1;
 				}
-				rc = ipmi_fru_set_field_string(intf, fru_id, *argv[3], *argv[4],
+				rc = ipmi_fru_set_field_string(intf, fru_id, *argv[3], atoi(argv[4]),  // the 4th param was changed to convert w/ atoi() by slash
```

Comment:
`atoi()` returns 0 on conversion errors. can't use it here. generally, try to avoid using `atoi()`, use `strtol()` instead.

Reply:
no need to put your name in a comment near each change. there is `git blame` for that.